Assets/__Scripts/MapEditor/Grid/Collections/NotesContainer.cs

```diff
@@ -11,7 +11,7 @@ public class NotesContainer : BeatmapObjectContainerCollection {
     [SerializeField] private TracksManager tracksManager;
 
     private HashSet<Material> allNoteRenderers = new HashSet<Material>();
-    private float epsilon = 0.001f;
+    public static float Epsilon = 0.001f;
```

Comment:
Because `BeatmapObject` is now using this, I think it'll make sense to move this to `BeatmapObjectContainerCollection`. Hopefully nothing should break from that, since this class inherits that anyways.